tests/data/UserFactory.ts

```diff
@@ -27,11 +27,13 @@ export class UserFactory {
   }
 
   public static fake(substitute?: Partial<UserModel>): UserModel {
+    const fName = faker.name.firstName();
+    const lName = faker.name.lastName();
     const fake = UserModel.create({
```

Comment:
nit: let's avoid variable naming shorthand and use `firstName` and `lastName`, even more so since we can simplify lines 34 and 35 to:
```ts
{
...
firstName,
lastName,
hash: UserFactory.PASSWORD_HASH,
...
}
```